part-4/example-workflows.md

```diff
@@ -0,0 +1,75 @@
+# MoBi - Exmple Workflows
```

Comment:
Added the Tumor extension example as a separate document - the minimal MoBi model can be also added here